docs/source/scenarios/container_inventory_management.rst

```diff
@@ -633,11 +633,86 @@ random mode, we hope this could help you learn how to use the environment interf
 
            # Drive the environment with the random action.
            metrics, decision_event, is_done = env.step(action)
-
+    
        # Query for the environment business metrics at the end of each episode,
        # it is usually users' optimized object in CIM scenario (usually includes multi-target).
        print(f"ep: {ep}, environment metrics: {env.metrics}")
        env.reset()
 
 Jump to `this notebook <https://github.com/microsoft/maro/tree/master/notebooks/container_inventory_management/interact_with_environment.ipynb>`_
 for a quick experience.
+
+
+Inter-view
+~~~~~~~~~~
+
+Generally speaking, the number of epochs in this scene will be greater
+than 100. In order to facilitate users to select specific data and
+observe the overall or partial data trend, the visualization tool
+provides data selection options in two dimensions.
+
+To change "Start Epoch" and "End Epoch", user could specify the selected
+data range. To change "Epoch Sampling Ratio", user could change the
+sampling rate of selected data, similar as Citi Bike `Citi Bike Examples <../citi_bike.html>`_
+
+For description of Attributes Selection in charts and Formula
+Calculation, please refer to Citi Bike `Citi Bike Examples <../citi_bike.html>`_
+
+.. figure:: ..\images\visualization\dashboard\cim_inter_epoch.gif
+   :alt: cim\_inter\_epoch
+
+Intra-view
+~~~~~~~~~~
+
+This part shows the data under a selected epoch. By scrolling the
+slider, users can select different epochs. Furthermore, this part of
+data is divided into two dimensions: by snapshot and by port according
+to time and space. In terms of data display, according to the different
+types of attributes, it is divided into two levels: accumulated data
+(accumulated attributes. e.g. acc\_fulfillment) and detail data.
+
+If user choose to view information by ports, attributes of the selected
+port would be displayed.
+
+Chart characteristics and data selection method please refer to `Citi Bike Examples <../citi_bike.html>`_
```

Comment:
It's better to introduce the general selection method in doc for visualization, not in citibike scenario?